e2e-tests/hardhat.config.ts

```diff
@@ -6,7 +6,7 @@ import "@nomiclabs/hardhat-ethers";
 
 const config: HardhatUserConfig = {
   zksolc: {
-    version: "1.5.7",
+    version: "1.5.8",
```

Comment:
Did not realize this was released yet 👀  Does this come with lib linking?

Reply:
Not sure, but the reason for this PR is that 1.5.8 only works with latest @matterlabs/hardhat-zksync-solc